src/ucc/api/ucc.h

```diff
@@ -151,16 +151,22 @@ typedef enum {
  *
  */
 typedef enum {
-    UCC_COLL_TYPE_BARRIER            = UCC_BIT(0),
-    UCC_COLL_TYPE_BCAST              = UCC_BIT(1),
-    UCC_COLL_TYPE_ALLREDUCE          = UCC_BIT(2),
-    UCC_COLL_TYPE_REDUCE             = UCC_BIT(3),
-    UCC_COLL_TYPE_ALLTOALL           = UCC_BIT(4),
-    UCC_COLL_TYPE_ALLGATHER          = UCC_BIT(5),
-    UCC_COLL_TYPE_GATHER             = UCC_BIT(6),
-    UCC_COLL_TYPE_SCATTER            = UCC_BIT(7),
-    UCC_COLL_TYPE_FANIN              = UCC_BIT(8),
-    UCC_COLL_TYPE_FANOUT             = UCC_BIT(9)
+    UCC_COLL_TYPE_ALLGATHER          = UCC_BIT(0),
+    UCC_COLL_TYPE_ALLGATHERV         = UCC_BIT(0),
```

Comment:
Why Allgather and Allgatherv have the same id?

Reply:
👍